src/Tile/RadioTile.svelte

```diff
@@ -20,24 +23,31 @@
   /** Specify a name attribute for the input */
   export let name = "";
 
+  /** Obtain a reference to the input HTML element */
+  export let ref = null;
+
   import { getContext } from "svelte";
   import CheckmarkFilled16 from "carbon-icons-svelte/lib/CheckmarkFilled16";
 
-  const { add, update, selectedValue } = getContext("TileGroup");
+  const { _light, add, update, selectedValue } = getContext("RadioTileGroup");
```

Comment:
Note: this is another breaking change if a RadioTile can no longer be used with a TileGroup

Reply:
Ah.

Reply:
What do you think to this feature? Support it?

Reply:
IMO this already deviates from the initial issue to improve the DX for SelectableTile better.

Reply:
ok